### PR TITLE
Fix a large ammount of formatting issues

### PR DESCRIPTION
**Pull request by FuriousLlama**
Warnings dropped from over 2k to around 50

### CODE REVIEW

File: backend/Directory.Build.props [hunk 1]
```diff
@@ -16,6 +16,24 @@
...
+            <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+            <PrivateAssets>all</PrivateAssets>
+        </PackageReference>
+    </ItemGroup>
+    <ItemGroup>
+        <PackageReference Include="Microsoft.Net.Compilers.Toolset" Version="4.1.0">
+            <PrivateAssets>all</PrivateAssets>
+            <IncludeAssets>runtime; build; native; contentfiles; analyzers;</IncludeAssets>
+        </PackageReference>
+    </ItemGroup>
```
Comment (asanchezr) [inline backend/Directory.Build.props:36]: I'm guessing this is to assist with linting / code formatting errors?

**Review by asanchezr — APPROVED** [review]

**Comment by devinleighsmith** [comment]
I'm not going to review all 750 files, but I'm assuming that you didn't do this by hand and so the chances of errors being introduced by any of these changes are low?

**Comment by FuriousLlama** [comment]
> I'm not going to review all 750 files, but I'm assuming that you didn't do this by hand and so the chances of errors being introduced by any of these changes are low?

That is correct. I mostly selected the errors that needed fixing vs the ones that should be dismissed. There _**should**_ be no logic changes

File: backend/Directory.Build.props (hunk 1 above)
Comment (FuriousLlama) [reply backend/Directory.Build.props:36]: That's correct, those packages where needed to get the analyzers working propertly

**Comment by devinleighsmith** [comment]
@FuriousLlama lets get this merged, as otherwise people will likely get conflicts with this.

**Comment by devinleighsmith** [comment]
Hey @FuriousLlama whats the plan with this one?

**Pull request closed without merging**